gravitee-node-api/src/main/java/io/gravitee/node/api/cluster/MemberListener.java

```diff
@@ -23,6 +23,4 @@ public interface MemberListener {
     void onMemberAdded(final Member member);
 
     void onMemberRemoved(final Member member);
-
-    void onMemberChanged(final Member member);
```

Comment:
Can you remember me what are the reasons behind the removal of `onMemberChanged` method?

Reply:
There is actually no way to detect a member changed...so this method wasn't used.